src/index.ts

```diff
@@ -1068,6 +1068,15 @@ export class ValidationService {
         const cb: ValidationEventCallback = async (event, callback) => {
             let validate = this.validators[uid];
             if (!validate) return true;
+            
+            if (
+                !input.dataset.valEvent &&
+                event && event.type === 'input' &&
+                !input.classList.contains(this.ValidationInputCssClassName)
+            ) {
```

Comment:
Minor nit, but I prefer the logical operators in a multi-line conditional to align on the left. Makes it easier to see all the component conditionals.

```suggestion
            if (!input.dataset.valEvent
                && event
                && event.type === 'input'
                && !input.classList.contains(this.ValidationInputCssClassName)
            ) {
```